test/Unit/Task/ShellTest.php

```diff
@@ -27,6 +27,7 @@ public function provideConfigurableOptions(): iterable
             [
                 'scripts' => [],
                 'triggered_by' => ['php'],
+                'triggered_by_filenames' => [],
```

Comment:
We'll also need a test-case that covers how the task behaves on the newly added configuration.